docs/components/popovers.md

```diff
@@ -255,16 +255,10 @@ Options can be passed via data attributes or JavaScript. For data attributes, ap
       <td>How popover is triggered - click | hover | focus | manual. You may pass multiple triggers; separate them with a space. `manual` cannot be combined with any other trigger.</td>
     </tr>
     <tr>
-      <td>constraints</td>
-      <td>Array</td>
-      <td>'hover focus'</td>
-      <td>An array of constraints - passed through to Tether. For more information refer to Tether's <a href="http://tether.io/#constraints">constraint docs</a>.</td>
-    </tr>
-    <tr>
       <td>offset</td>
-      <td>string</td>
-      <td>'0 0'</td>
-      <td>Offset of the popover relative to its target. For more information refer to Tether's <a href="http://tether.io/#offset">offset docs</a>.</td>
+      <td>number</td>
+      <td>0</td>
+      <td>Offset of the popover relative to its target. For more information refer to Popper's <a href="https://popper.js.org/popper-documentation.html#new_Popper_new">offset docs</a>.</td>
```

Comment:
It accepts both a number or a string as explained here:
https://popper.js.org/popper-documentation.html#Modifiers.offset